README.md

```diff
@@ -13,85 +13,72 @@
 
-![envd](./docs/images/envd.png)
+![envd](https://user-images.githubusercontent.com/5100735/188821980-dcbd9069-b504-436a-9ffd-05ac5543a6d1.png)
 
 [^1]: The build language is [starlark](https://docs.bazel.build/versions/main/skylark/language.html), which is a dialect of Python.
 
-⏱️ **Save you plenty of time**
+⏱️ **6x faster build**
 
-`envd` adopts a multi-level cache mechanism to accelerate the building process. For example, the PyPI cache is shared across builds and thus the package will be cached if it has been downloaded before. It saves plenty of time, especially when you update the environment by trial and error.
+`envd` adopts a multi-level cache mechanism to accelerate the building process. For example, the PyPI cache is shared across builds and thus the package will be cached if it has been downloaded before. It saves plenty of time, especially when you update the environment by trial and error.[^2]
 
-<table>
-<tr>
-<td> envd </td> <td>
+<p align=center>
+  <img src="https://user-images.githubusercontent.com/5100735/188601795-8c37f5a3-b13b-422b-816f-8a0c51f1f8b1.svg" width="65%"/>
+</p>
 
-Docker[^2]
+[^2]: Docker without [buildkit](https://github.com/moby/buildkit)
 
-</td>
+☁️ **Local & cloud native**
+
+<table>
+<tr>
+<td>Local development simplifies the debugging, but...</td> 
+<td>Setup local & cloud native environment with envd</td>
 </tr>
 <tr>
 <td>
 
-```diff
-$ envd build
-=> pip install tensorflow       5s
-+ => Using cached tensorflow-...-.whl (511.7 MB)
-```
+❌ Can be complex to setup. When they break, you often need to run the whole setup.
 
-</td>
-<td>
+❌ Resource intensive because of the constraints to your CPU, moemory and local GPU.
 
-```diff
-$ docker build
-=> pip install tensorflow      278s
-- => Downloading tensorflow-...-.whl (511.7 MB)
-```
+❌ Cannot be reproducible. Because not everyone has an identical setup.
 
 </td>
-</tr>
-</table>
-
-[^2]: Docker without [buildkit](https://github.com/moby/buildkit)
-
-☁️ **Local & cloud native**
-
-`envd` integrates seamlessly with Docker, you can share, version, and publish `envd` environments with Docker Hub or any other OCI image registries. The `envd` environments can be run on Docker or Kubernetes.
+<td>
 
-🔁 **Repeatable builds & reproducible results**
+✅ AI infrastructure as code, reproduce the environment painlessly.
 
-You can reproduce the same dev environment, on your laptop, public cloud VMs, or Docker containers, without any change in setup. You can also collaborate with your colleagues without "let me configure the environment in your machine".
+✅ Separate your environment to avoid impacting your local configuration.
 
-🖨️ **Seamless experience of Jupyter/VSCode**
+✅ Use larger or more specialized hardware.
```

Comment:
```suggestion
✅ Use larger or more specialized hardwares.
```

Reply:
The noun hardware is uncountable, I think.